trunk/SUAVE/Methods/Weights/Correlations/BWB/payload.py

```diff
@@ -0,0 +1,55 @@
+# payload.py
```

Comment:
There are a lot of variables that are present in the T&W configuration which are not present in the BWB. For example, comparing the "systems.py" files for the two correlations, I took out the inclusion of the weight of elevators from the BWB configuration.  Some file however, such as the " wings_main.py"  and "landing_gear.py" correlation files are identical.